.circleci/local_publish_helpers.sh

```diff
@@ -279,3 +279,18 @@ function runE2eTest {
         yarn run e2e --force-exit --detectOpenHandles --maxWorkers=3 $TEST_SUITE
     fi
 }
+
+function checkPackageVersionsInLocalNpmRegistry {
+    cli_internal_version=$(npm view @aws-amplify/cli-internal version)
+    cli_version=$(npm view @aws-amplify/cli version)
+
+    echo "@aws-amplify/cli-internal version: $cli_internal_version"
+    echo "@aws-amplify/cli version: $cli_version"
+
+    if [[ $cli_internal_version != $cli_version ]]; then
+        echo "Versions did not match."
+        exit 1
+    else
+        echo "Versions matched."
```

Comment:
looks good, but usually if we fail we provide resolution steps if possible, are there any resolution steps available?

Reply:
I agree with @awsluja . 
Well one way is to update this file https://github.com/aws-amplify/amplify-cli/blob/2b81f92bb5d53b4da1705c47fdf8035c26be3caa/packages/amplify-cli-npm/index.ts#L19

Can we have some logic in ```cli-npm```  package that can fetch correct hash of the cli-internal package published ?

Reply:
The scope of this PR is to catch mismatch in PR checks, i.e. shorten feedback loop. not to develop fully automatic solution which is more expensive.

The resolution step today is to create PR that has proper conventional commit message and touches either cli or cli-internal depending on which was left behind.

Reply:
We can update this message in script